CONTRIBUTING.md

```diff
@@ -10,41 +10,91 @@ in the community are more likely to value and accept your contributions.
+  notifications when things change. This will help you remain aware.
+* Most of your communication (say 80%) should be through the issue tracker and
+  issue boards (called "projects" in GitHub). Use them to report problems,
+  propose features, strategize, give feedback, propose changes to policies
+  and procedures, ask questions, offer changes through merge requests,
+  review changes, etc. This is this preferred way to communicate as it
+  allows us to organize conversations around issues and because it is public.
+* Some of your communication (say 19%) should be through [#TODO chat system]
+  to ask questions, schedule meetings, hold meetings,
+  give or receive live help, give or receive demos or tutorials,
+  socialize, etc.
+* A minimum of your of your communication (say 1% or less)
+  should be through email. Only use email for personal, private matters.
+
+There are several FOSSisms at play here:
+
+* FOSSism #1: It's all about community
+* FOSSism #5: If it isn't public, it didn't happen
+* FOSSism #6: Embrace radical transparency
+* FOSSism #9: Keep a history
+* FOSSism #16: Avoid uncommunicated work
+
+We prefer public issue trackers over email because it is public
+and transparent. We prefer issue trackers over chat communication because
+it is more organized.
+
+For more information about our issue trackers, issue boards, issues, and labels
+please read [Issue Tracker](issue-tracker.md).
 
 ## Initial setup
 
-* Install and configure Git
-    * Choose a name and email that you are comfortable with being public that will be use to credit you with your work.
-* Create a GitLab account
-    * Use the same email as you did for configuring Git. This will allow GitLab to link your commits to your account.
-    * Fork the upstream project on GitLab. If you are working in a shop, your shop manager will create a shop fork. Use this instead of creating your own. Your shop manager needs to give you permissions to push to the shop fork.
+At a minimum __create a GitHub and GitLab account__. This will allow you to
+create and comment on issues. Choose a name and email that you are
+comfortable with being public. GitHub and GitLab will identify you by
+this name and email when you make and comment on issues.
+They will also use this information to link your commits to your account.
+
+If you intend to make changes you will need to __install and configure Git__.
+
+* Download and install git from [git-scm.com](https://git-scm.com/).
+* [Configure Git](https://git-scm.com/book/en/v2/Getting-Started-First-Time-Git-Setup).
+
+When you configure Git, give the same name and email that you gave to GitHub
+and GitLab. Again, remember this name and email will become public, as git will
+include them in each commit you make, and when you contribute them, they
+will be publicly visible.
+
+Next, on GitHub or GitLab, __fork the project(s)__ you will be working on.
+If you are working in a shop, your shop manager will create a shop fork.
```

Comment:
Maybe link to the shop setup documentation somewhere under here.

Reply:
Good idea. I'm adding it.